src/index.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
+import App from './App/App';
```

Comment:
Why did you change the file format for this?

Reply:
So as the project grows the file trees make sense, for example; the components being in the component folder. than an App folder for the Top level where everything will get called from.

Reply:
Typically you would have the folder be App/index.tsx. That way you don't have to change the import. We can talk a little about this today if you want.